### PR TITLE
Rankings page info icon

### PR DESCRIPTION
**Pull request by saiprakash-v**
Trello Link: https://trello.com/c/YJQ0vGpR
##### SUMMARY

Tooltip will be shown on clicking the info icon in rankings page

### CODE REVIEW

File: custom/icds_reports/static/icds_reports/mobile/css/map-chart.less [hunk 1]
```diff
@@ -9,6 +9,7 @@
   .p-3;
   .rounded-border;
   max-height: ~"calc(100vh - 170px)";
+  height: ~"calc(100vh - 170px)";
```
Comment (czue) [inline custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: why was this necessary? and is it now redundant with `max-height`?

**Review by czue — COMMENTED** [review]
Can you attach a screenshot? Also, any reason you didn't use the mobile slide-up UI for this?

(waiting on approval till screenshot and responses)

File: custom/icds_reports/static/icds_reports/mobile/css/map-chart.less (hunk 1 above)
Comment (saiprakash-v) [reply custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: height in styling defines height of the element when its child element height is less or equal to the parent height. If child element height goes beyond parent height, parent height matches with child height. So max height is set to prevent parent height from increasing.

Comment (saiprakash-v) [reply custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: Screenshot from 2020-01-17 15-02-01
This is how it looks when clicked on 'i'. Saket mentioned to keep it the way it is in web

Comment (saiprakash-v) [reply custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: Screenshot from 2020-01-17 15-13-40
If height is removed and when there is no data. it looks this way

Comment (czue) [reply custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: I see. So it's kind of an unrelated change? 

I actually prefer the section only growing to take the needed space. Perhaps if the issue is clicking the "i" with no data, you could just set a `min-height`?

Comment (saiprakash-v) [reply custom/icds_reports/static/icds_reports/mobile/css/map-chart.less]: sure.
